kong/llm/state.lua

```diff
@@ -0,0 +1,97 @@
+local _M = {}
+
+-- Set disable_ai_proxy_response_transform if response is just a error message or has been generated
+-- by plugin and should skip further transformation
+function _M.disable_ai_proxy_response_transform()
+  kong.ctx.shared.llm_disable_ai_proxy_response_transform = true
+end
+
+function _M.should_disable_ai_proxy_response_transform()
+  return kong.ctx.shared.llm_disable_ai_proxy_response_transform == true
```

Comment:
Just a tiny suggestion: we can just `return kong.ctx.shared.llm_disable_ai_proxy_response_transform` since we ensure it is a bool itself by a dedicated function.

Reply:
Good point, we can fix in the next round of refactor.